src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -168,6 +168,10 @@
     "DESCRIPTION": "stsjorbsmod:Entombed. stsjorbsmod:Ephemeral. NL stsjorbsmod:Exhume whenever an enemy dies. NL Deal !D! damage. NL Remember stsjorbsmod:Wrath.",
     "UPGRADE_DESCRIPTION": "stsjorbsmod:Entombed. stsjorbsmod:Ephemeral. NL stsjorbsmod:Exhume whenever an enemy dies. NL Deal !D! damage. NL Gain stsjorbsmod:Clarity of stsjorbsmod:Wrath."
   },
+  "stsjorbsmod:DemonicCoup": {
+    "NAME": "Demonic Coup",
+    "DESCRIPTION": "Gain !M! Ritual. NL stsjorbsmod:Exert."
```

Comment:
I think this is missing a `stsjorbsmod:Ephemeral. NL ` at the start (the ephemeral effect is implemented correctly, just missing on the text)

Reply:
Yup, you're right, I'll add.